### PR TITLE
Authorized => Authenticated

### PR DESCRIPTION
**Pull request by HaoK**
@Rick-Anderson @rachelappel @blowdart

### CODE REVIEW

File: aspnet/security/authorization/introduction.rst [hunk 1]
```diff
@@ -15,4 +15,4 @@ In ASP.NET Core authorization now provides simple declarative :ref:`role <securi
 Namespaces
 ----------
 
-The authorization attribute is part of the MVC namespace, specifically you must add  ``using Microsoft.AspNetCore.Authorization;``
+The authorization attribute is part of the Microsoft.AspNetCore.Authorization namespace, specifically you must add  ``using Microsoft.AspNetCore.Authorization;``
```
Comment (Rick-Anderson) [inline aspnet/security/authorization/introduction.rst]: The authorization attribute is in the ``Microsoft.AspNetCore.Authorization`` namespace.

Comment (blowdart) [inline aspnet/security/authorization/introduction.rst]: Why not "The `[Authorize]` attribute is part of", probably easier to understand?

File: aspnet/security/authorization/simple.rst [hunk 2]
```diff
@@ -3,7 +3,7 @@
 Simple Authorization
 ====================
 
-Authorization in MVC is controlled through the ``Authorize`` attribute and its various parameters. At its simplest applying the ``Authorize`` attribute to a controller or action limits access to the controller or action to any authorized user.
+Authorization in MVC is controlled through the ``Authorize`` attribute and its various parameters. At its simplest applying the ``Authorize`` attribute to a controller or action limits access to the controller or action to any authenticated user.
```
Comment (blowdart) [inline aspnet/security/authorization/simple.rst]: Hah, good catch.

File: aspnet/security/authorization/introduction.rst (hunk 1 above)
Comment (HaoK) [inline aspnet/security/authorization/introduction.rst]: Andrew actually updated the wording in his PR as well, I'm going to close this one and use his wording, and make the below change in the new PR

Comment (HaoK) [inline aspnet/security/authorization/introduction.rst]: Actually this fix was already merged in another PR, so we should ignore this change for this PR, its only the lower simple one

**Comment by Rick-Anderson** [comment]
@HaoK  can you fix the merge conflicts and merge this?

**Comment by HaoK** [comment]
Replaced by https://github.com/aspnet/Docs/issues/1477 which has the rebased PR

**Pull request closed without merging**